### PR TITLE
Add new author to citation

### PR DESCRIPTION
**Pull request by ziw-liu**
Add @alishbaimran to CITATION.cff.

### CODE REVIEW

**Comment by ziw-liu** [comment]
@alishbaimran is this you?

https://orcid.org/0009-0004-1133-0408

**Comment by alishbaimran** [comment]
@ziw-liu this is me: https://orcid.org/0009-0003-7049-355X

File: CITATION.cff [hunk 1]
```diff
@@ -12,23 +12,27 @@ authors:
...
     affiliation: Chan Zuckerberg Biohub San Francisco
-    orcid: 'https://orcid.org/0000-0002-2542-3582'
-repository-code: 'https://github.com/mehta-lab/VisCy'
-url: 'https://github.com/mehta-lab/VisCy'
+    orcid: "https://orcid.org/0000-0002-2542-3582"
+repository-code: "https://github.com/mehta-lab/VisCy"
+url: "https://github.com/mehta-lab/VisCy"
 abstract: computer vision models for single-cell phenotyping
 keywords:
   - machine-learning
```
Comment (mattersoflight) [inline CITATION.cff:38]: - image translation
- representation learning
- cell state segmentation

**Review by mattersoflight — APPROVED** [review]

File: CITATION.cff (hunk 1 above)
Comment (ziw-liu) [reply CITATION.cff:38]: Also these keywords are software identifiers more than research identifiers and were taken from existing GitHub topics. I propose only adding `image-translation` and `representation-learning` here.

Comment (ziw-liu) [reply CITATION.cff:38]: See https://github.com/mehta-lab/VisCy/pull/188/commits/bc6c4561329c669d854f24395e96df7760778a24. I also updated the GitHub topics.

**Pull request merged**